package.json

```diff
@@ -13,22 +16,24 @@
     "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
+    "autoprefixer": "^9",
```

Comment:
This is a weird syntax, should it be ^9.0.0?

Reply:
I think this is a yarn quirk. weird